AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Subscribe/DataStorePublisher.swift

```diff
@@ -8,16 +8,13 @@
 import Amplify
 import Combine
 
-// TODO: Should this be a multicast publisher?
 @available(iOS 13.0, *)
-struct DataStorePublisher: DataStoreSubscribeBehavior {
+struct DataStorePublisher: ModelSubcriptionBehavior {
 
     private let subject = PassthroughSubject<MutationEvent, DataStoreError>()
 
-    func publisher<M: Model>(for modelType: M.Type) -> AnyPublisher<MutationEvent, DataStoreError> {
-        return subject
-            .filter { $0.modelName == modelType.modelName }
-            .eraseToAnyPublisher()
+    func publisher() -> AnyPublisher<MutationEvent, DataStoreError> {
+        return subject.eraseToAnyPublisher()
```

Comment:
Looking at this again: We've been modeling `publisher` as a property (usually computed) rather than a method.

Reply:
Moved to computed property